configure.ac

```diff
@@ -613,6 +614,11 @@ MOTR_SEARCH_LIBS([pthread_create], [c pthread], [PTHREAD_LIBS],
 )
 AC_SUBST([PTHREAD_LIBS])
 
+MOTR_SEARCH_LIBS([MD5_Init], [c crypto], [OPENSSL_LIBS],
```

Comment:
Also, I have a question here.
library "crypto" is included in openssl-devel packages?

Do we need the "c" library?

Reply:
Hua, I have not done integration before. I copied code from Sanjog's changes. Sanjog has looked at this code.